internal/config/profile.go

```diff
@@ -641,19 +666,32 @@ func (p *Profile) Save() error {
 	return viper.WriteConfigAs(p.Filename())
 }
 
+// OldMongoCLIConfigHome retrieves configHome path based used by mongoCLI.
+func OldMongoCLIConfigHome() (string, error) {
+	if home := os.Getenv("XDG_CONFIG_HOME"); home != "" {
+		return "", errors.New("not applicable")
+	}
+
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("%s/.config", home), nil
+}
+
```

Comment:
Could you add deprecatation comments saying that mongocli version prior v1.24.0 uses this path and that new versions use `MongoCLIConfigHome`? 

I'd also rename the function as `DeprecatedMongoCLIConfigHome` but this is just my preference

Reply:
> add a paragraph to its doc comment that begins with “Deprecated:” followed by some information about the deprecation.

https://go.dev/blog/godoc